image/gimp_brush.ksy

```diff
@@ -0,0 +1,43 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
```

Comment:
```suggestion
  file-extension: gbr
  xref:
    mime: image/x-gimp-gbr
```

according to GIMP source code (https://github.com/GNOME/gimp/blob/9b6d59f/app/core/gimpbrush-load.c#L283):

```c
"mime-type", "image/x-gimp-gbr",
```